frontend/app/components/comment/_level/comment_level.scss

```diff
@@ -1,17 +1,10 @@
-$step: 24px;
-
-@for $i from 1 to 5 {
-  .comment_level {
-    &_$(i) {
-      padding-left: calc($i * $step);
-
-      &.comment_replying {
-        .comment__input {
-          @media (-moz-touch-enabled: 1) and (max-width: 768px), (pointer: coarse) and (max-width: 768px) {
-            margin-left: calc(-$i * $step);
-          }
-        }
-      }
-    }
-  }
+.thread > .thread {
+  padding-left: 17px;
+  border-left: 1px solid rgba(0, 0, 0, 0.15);
```

Comment:
Is it me, or lines color is too active in light theme?

![Screenshot 2019-06-30 at 02 14 01](https://user-images.githubusercontent.com/884649/60390322-d5aedc00-9adc-11e9-856b-760dfd444e25.png)

I'd rather mute it to rgba(0,0,0,.07)
![Screenshot 2019-06-30 at 02 17 33](https://user-images.githubusercontent.com/884649/60390341-4d7d0680-9add-11e9-83a1-51e578a3f309.png)

... or even to rgba(0,0,0,.05)
![Screenshot 2019-06-30 at 02 14 15](https://user-images.githubusercontent.com/884649/60390320-d5164580-9adc-11e9-8eb1-45ae2fb65e5f.png)

... or another variant to make border dotted:
![Screenshot 2019-06-30 at 02 13 29](https://user-images.githubusercontent.com/884649/60390321-d5164580-9adc-11e9-898e-0c94c8a1eb32.png)